cli/lib/kontena/cli/apps/service_generator_v2.rb

```diff
@@ -0,0 +1,23 @@
+require 'yaml'
+require_relative 'service_generator'
+
+module Kontena::Cli::Apps
+  class ServiceGeneratorV2 < ServiceGenerator
+
+    def parse_data(options)
+      data = super(options)
+      data['net'] = options['network_mode'] if options['network_mode']
+      data['log_driver'] = options['logging']['driver'] if options['logging'] && options['logging']['driver']
+      data['log_opts'] = options['logging']['options'] if options['logging'] && options['logging']['driver'] && !options['logging']['driver'].empty?
```

Comment:
Would it make these more readable if `dig`is used instead on key access nesting?